roles/repositories/tasks/from_file.yml

```diff
@@ -0,0 +1,43 @@
+- name: Create Full Paths
+  ansible.builtin.set_fact:
+    repos_file_list: "{{ repos_file_list + [([repos_directory, item] | path_join) | string] }}"
+  loop: "{{ repositories_file_names }}"
```

Comment:
what is this variable? Different from `repos_file_list`?

Reply:
Users will use which one?

Reply:
Users will use the variable described in readme. Other one is a meta variable, which contains full path and should not be edited by user. We may rename those forbidden or reserved variables by starting with underscore, for example.